assembly/src/docs/20_INSTALL_WINDOWS_WITH_GRAALVM.md

```diff
@@ -0,0 +1,133 @@
+# Build TornadoVM with Graal 20.2.0 JDK 8 / 11 on Windows x64
+## Install tools
+### Maven
+Download Apache Maven from the [official site](https://maven.apache.org/download.cgi) and extract it to the any location. Below it's assumed that Maven's home is `C:/Maven`.
+### MSys2
+1. Download [MSys2](https://www.msys2.org/) installer from the official site and run it. You may choose any installation directory, below it will be referred as `<MSYS2>`. IMPORTANT: the only executable you should use as a terminal is `<MSYS2>/mingw64.exe`.
+
+2. Update MSys2 _**system**_ packages. Start `<MSYS2>/mingw64.exe` and run the following command in the terminal:
+```bash
+pacman -Syu
+```
+You might need to execute it several times until you see that no updates found.
+
+3. Update MSys2 _**default**_ packages. In the terminal window of `<MSYS2>/mingw64.exe` run:
+```bash
+pacman -Su
+```
+You might need to execute it several times until you see that no updates found.
+
+4. Install necessary tools to MSys2. In the terminal window of `<MSYS2>/mingw64.exe` run:
+```bash
+pacman -S \
+mingw-w64-x86_64-make           \
+mingw-w64-x86_64-cmake          \
+mingw-w64-x86_64-gcc            \
+mingw-w64-x86_64-opencl-headers \
+mingw-w64-x86_64-opencl-icd-git \
+python make git
+```
+
+5. Create default Maven repository for MSys2 user:
+```bash
+cd ~
+mkdir .m2
+```
+
+6. Create file `mvn` in `<MSYS2>/mingw64/bin` with any textual editor according to the template below:
+```bash
+#!/usr/bin/env bash
+BASH_FILE=$(where bash)
+SETTINGS=\"${BASH_FILE%\\*\\*\\*}${HOME}/.m2/settings.xml\"
+
+C:/Maven/bin/mvn.cmd --settings ${SETTINGS} "$@"
+```
+You should only change path to Maven executable to the actual location of Maven in your system.
```

Comment:
Such "proxy" is necessary - otherwise Maven will not see `settings.xml` - and requesting user to specify settings' file location is not an option here.

Reply:
In my case, it works without the custom `mvn.cmd`. But I am ok to keep it in this PR.